components/Collapsible/Collapsible.js

```diff
@@ -0,0 +1,92 @@
+import { useState } from "react";
+import styled from "styled-components";
+import ExerciseForm from "../ExerciseForm/ExerciseForm";
+
+
+const StyledContainer = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: ${({ expand }) => (expand ? "column" : "row")};
+  align-items: center;
+  position: relative;
+`;
+
+const StyledParameters = styled.div`
+  margin: 0 0.8rem;
+`;
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 40%;
+  margin-top: 1rem;
+`;
+
+const StyledExpandControl = styled.span`
+  position: absolute;
+  top: 0;
+  right: 0;
+
+  &:hover {
+    cursor: pointer;
+  }
+`;
+
+export default function Collapsible({
+  id,
+  sets,
+  reps,
+  weight,
+  dayId,
+  handleUpdateExercise,
+}) {
```

Comment:
This component does much more than just handling collapsible content. Consider splitting this into multiple parts (Collapsible, ExerciseDetail, etc.)

Reply:
Thanks. I'll look into it.